pkgs/by-name/py/pyflyby/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "pyflyby";
+  version = "1.9.6";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "deshaw";
+    repo = "pyflyby";
+    rev = version;
+    hash = "sha256-QkoFr9tFtZ+ZEWlxe9csrzoYFl9/V2l4hKYfUWsXUdc=";
+  };
+  disabled = python3.pythonOlder "3.8";
+
```

Comment:
```suggestion
```
The minimum Python version in nixpkgs is 3.9 so this expression wouldn't do much.

Reply:
I was following upstream's restriction.

Reply:
Sorry if I was unclear; I wasn't asking you to not follow upstream's restriction, I simply pointed out that in nixpkgs that restriction is implied in this case (and is even stricter).